be/src/runtime/current_thread.h

```diff
@@ -152,7 +156,10 @@ class CurrentThread {
     const static int64_t BATCH_SIZE = 2 * 1024 * 1024;
 
     int64_t _cache_size = 0;
+    // Store in TLS for diagnose coredump easier
     TUniqueId _query_id;
+    TUniqueId _fragment_instance_id;
```

Comment:
Do we care the fragment instance id?

Reply:
I'm not sure. But more information could reduce the effort when diagnosing.